src/pybind/mgr/cephadm/migrations.py

```diff
@@ -12,7 +12,7 @@
 if TYPE_CHECKING:
     from .module import CephadmOrchestrator
 
-LAST_MIGRATION = 5
+LAST_MIGRATION = 6
```

Comment:
Thinking. There was no production cluster with this the agent enabled. IMO we can avoid a migration here. We don't support migrations between development version.

Reply:
alright, I'll take that bit out